indico/modules/rb_new/client/js/modules/bookRoom/BookRoom.jsx

```diff
@@ -309,78 +309,26 @@ class BookRoom extends React.Component {
 
         return (
             <>
-                <Header as="h2">
-                    <Translate>Rooms that you might be interested in</Translate>
+                <Divider horizontal>
+                    <Icon name="magic" style={{fontSize: '2em'}} circular />
+                </Divider>
+                <Header style={{margin: '0 0 2em 0'}}>
+                    <Translate>Here are some other possibilities we've calculated for you!</Translate>
```

Comment:
"calculated" sounds a bit strange IMO - I've never seen that term used for any kind of recommendation on websites